lib/main_common.pm

```diff
@@ -2029,32 +2029,6 @@ sub load_x11_webbrowser {
     }
 }
 
-sub load_x11_message {
-    if (check_var("DESKTOP", "gnome")) {
-        loadtest "x11/empathy/empathy_irc"       if is_sle("<15");
```

Comment:
`empath_irc` is not added in yaml schedule?

Reply:
I checked this against sles 15 sp4, and empathy is not available at all. So drop this at first and open a ticket for it.

please see my verification run: http://10.160.64.152/tests/3344#step/empathy_irc/8

Reply:
and this has been already dropped: http://openqa.suse.de/tests/7029674#